### DISCUSSION TITLE
Can not pass the Teacher proof check

### DISCUSSION THREAD

**Discussion by Siunaus** [community/community#48678] [GitHub Education]
I took photos for my employee card and translation documents and the system alway tell me.

 You are significantly more likely to be verified if you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

I have update my profile in English or in Chinese.  And it's not work

**Comment by Siunaus** [comment]
I will try it every day when I am free until it take my proof

**Comment by samus-aran** [comment]
Hi @Siunaus 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process. If your original document or profile information is not in English then we advise you to take a photo of the original next to an English translation or update your profile info to be in English as much as possible.

Let us know how you get on 👍

**Reply by Siunaus** [reply]
First of all, thank you for your reply and help. The fact is that I work in a branch of a university and the school I choose is not in the same city. In this case, can I provide an efficient employment document and its translation to pass the verification?

I understand the necessity of your review, but for college teachers, the review process is somewhat unreasonable
